evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -1938,11 +1945,13 @@ where
                     dummy_circuit(&table_circuit.circuit.common);
                 let dummy_pis = HashMap::new();
                 let dummy_proof = dummy_proof(&dummy_circuit, dummy_pis)
-                    .expect("Unable to generate dummy proofs");
+                    .map_err(|_| anyhow::format_err!("Unable to generate dummy proofs"));
```

Comment:
I just realized it now, but this isn't negligible I think (light, but not negligible), as we're including build time + dummy proof gen every time. Ideally have this proof generated during pre-processing and reused (especially as we may need this often following extension to other tables).

Reply:
> I just realized it now, but this isn't negligible I think (light, but not negligible), as we're including build time + dummy proof gen every time. Ideally have this proof generated during pre-processing and reused (especially as we may need this often following extension to other tables).

Done